<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>0</number>
     </property>
     <widget class="QGroupBox" name="side_bar">
      <property name="maximumSize">
       <size>
        <width>233</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{
background-color: #171717;
border:none;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="logo_block">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>65</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>65</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox{
border:none;
background-transparent;
border-bottom: 1px solid rgba(255,255,255,100);
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="logo_label">
              <property name="minimumSize">
               <size>
                <width>33</width>
                <height>33</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>33</width>
                <height>33</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../bot_icons.qrc">:/icons/widgets/times_icon/logo.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="naming_label">
              <property name="font">
               <font>
                <family>Futura LT</family>
               </font>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;Simple&lt;/span&gt;&lt;span style=&quot; font-size:24pt; color:#4d96ff;&quot;&gt;.gram&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">QLabel{
padding-left:10px;
color: rgba(255,255,255,100);}</string>
           </property>
           <property name="text">
            <string>Bot systems:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ProjectsListWidget" name="projects_list">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QListWidget{
border:none;
background:transparent;

}
QListWidget::item{
padding-left:20px;
border:none;
color: rgba(255,255,255,100);
}
QListWidget::item:hover{
border-left:2px solid rgba(255,255,255,180);
color: rgba(255,255,255,180);
}
QListWidget::item:selected{
border:none;
color: rgb(255,255,255);
background: rgba(255,255,255,25);
}</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="new_project_button">
           <property name="styleSheet">
            <string notr="true">QPushButton{
border:none;
background-style:transparent;
color: #4DAAFF;
padding-left:20px;
text-align: left;
}</string>
           </property>
           <property name="text">
            <string>+ New Project</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="settings_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:none;
background:transparent;
color:rgba(255,255,255,100);
border-top:1px solid rgba(255,255,255,100);
padding-left:20px;
text-align: left;
}
QPushButton::hover{
color:rgba(255,255,255,180);
}
QPushButton::pressed{
color:rgb(255,255,255);
}</string>
         </property>
         <property name="text">
          <string>? Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="centrall_panel">
      <property name="styleSheet">
       <string notr="true">QGroupBox{
border:none;
background: rgb(241,241,241);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="top_pannel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>65</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>65</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox{
background-color: #FFFFFF;
boder:none;
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="user_widget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox{
background-color: #171717;
color:white;
border-radius:10px;
border:none;
}

QComboBox::hover{
border:white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     /*background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);*/
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
   /*padding-top: 3px;
   padding-left: 4px;*/
}

QComboBox::drop-down {
    background-color:transparent;
	border:none;
}

QComboBox::down-arrow {
    /*image: url(/usr/share/icons/crystalsvg/16x16/actions/1downarrow.png);*/
	background-color: white;
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
            </property>
            <item>
             <property name="text">
              <string>User</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="centrall_pannel_widget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QSteckedWidget{
border:none;
background: rgb(241,241,241);
}</string>
         </property>
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="LoginWidget" name="loginWindow">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="mainWindow">
          <layout class="QVBoxLayout" name="verticalLayout_5"/>
         </widget>
         <widget class="SelectedProjectWidget" name="botShowWindow"/>
         <widget class="AddNewProjectWidget" name="botNewCreator">
          <layout class="QHBoxLayout" name="horizontalLayout_3"/>
         </widget>
         <widget class="QWidget" name="botRedactorWindow"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginWidget</class>
   <extends>QWidget</extends>
   <header>constructor_app.widgets.login_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AddNewProjectWidget</class>
   <extends>QWidget</extends>
   <header>constructor_app.widgets.add_new_project_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectedProjectWidget</class>
   <extends>QWidget</extends>
   <header>constructor_app.widgets.selected_project_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProjectsListWidget</class>
   <extends>QListWidget</extends>
   <header>constructor_app.widgets.projects_list_widget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bot_icons.qrc"/>
 </resources>
 <connections/>
</ui>
